packages/ingest/queue.js

```diff
@@ -85,3 +89,41 @@ async function enqueueGranuleIngestMessage(
   return sendSQSMessage(queueUrl, message);
 }
 exports.enqueueGranuleIngestMessage = enqueueGranuleIngestMessage;
+
+/**
+ * Queue a workflow to be picked up by SF starter
+ *
+ * @param {Object} event - event to queue with workflow and payload info
+ * @returns {Promise} - resolves when the message has been enqueued
+ */
+async function queueWorkflowMessage(event) {
+  const template = get(event, 'template');
+  const provider = get(event, 'provider', {});
+  const collection = get(event, 'collection', {});
+  const payload = get(event, 'payload', {});
+
+  const message = await getMessageFromTemplate(template);
+
+  let queueUrl = null;
+
+  if (message.resources) {
```

Comment:
Is there documentation around when this is message.resources and when it's message.meta?

Reply:
That was from when I ported it over and never removed it. So I removed that entire block.